List-Comprehensions.ipynb

```diff
@@ -95,8 +95,13 @@
    "cell_type": "code",
```

Comment:
<p>Let's use <strong>list comprehensions in</strong> some examples:</p><p> </p><p>Typo: Example 4: enumerate within list<strong><s>s</s></strong> comprehensions</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neuefische/ds-meetups/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>